src/components/ChatBubble/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useRecoilValue } from "recoil";
+import { ChatBubbleContainer, ChatBubbleHoverContent } from "./index.element";
+import { SetStateAction, useState } from "react";
+import { talkState } from "@/states";
+
+interface IChatBubbleProps {
+  from: 1 | 2;
+  text: string;
+  time: string;
+  liked: boolean;
+  setLiked: any;
+}
+
+const ChatBubble = ({
+  from,
+  text,
+  time,
+  liked,
+  setLiked,
+}: IChatBubbleProps) => {
+  const talker = useRecoilValue(talkState);
+  const [isHovered, setIsHovered] = useState(false);
+
+  const handleClickBubble = (e: React.MouseEvent<HTMLDivElement>) => {
+    if (from === 2) {
+      if (e.detail >= 2) {
+        setLiked();
+      } else {
+        setIsHovered(true);
+      }
+    }
+  };
+
+  return (
+    <div style={{ display: "flex", flexDirection: "column" }}>
+      <ChatBubbleContainer
+        from={from === 1 ? "me" : "partner"}
+        liked={liked}
+        onClick={(e) => handleClickBubble(e)}
```

Comment:
```suggestion
        onClick={handleClickBubble}
```

이벤트 객체만 받는 건 파라미터 생략해주는 것도 좋을 것 같아요